src/sentry/api/serializers/models/alert_rule_trigger_action.py

```diff
@@ -8,18 +8,28 @@ def human_desc(self, action):
         # Returns a human readable description to display in the UI
         if action.type == action.Type.EMAIL.value:
             if action.target:
-                if action.target_type == action.TargetType.USER.value:
-                    return "Send an email to " + action.target.email
+                if action.target_type == action.TargetType.SPECIFIC.value:
+                    return "Send an email to " + action.target, action.target
```

Comment:
We don't support this option currently, let's not add it.

Reply:
It was mainly to fix a test but I think the test was written incorrectly, I'll change it